readme.md

```diff
@@ -80,6 +80,21 @@ See examples.
 
 #### nextQuery(): string;
 
+#### makeIterable(resp?: Object): AsyncIterable<Object>
+
+<details>
+
+```js
+const json = await pixiv.searchIllust('艦これ10000users入り')
+let ar = []
+for await (const r of pixiv.makeIterable(json)){
+  ar = ar.concat(r.illusts)
```

Comment:
Nice work 👍  However, this will get you into API limits. A delay is required for each request.

```js
for await (const r of pixiv.makeIterable(json)){
  ar = ar.concat(r.illusts)
  await delay(1000) // avoid API limitations
}
```

Reply:
```js
const first = await pixiv.userBookmarksIllust(pixiv.auth.user.id)
let ar = []
for await (const r of pixiv.makeIterable(first)){
  ar = ar.concat(r.illusts)
}
console.log(ar.length)
```
I tried this code on my account, and it correctly gave me the number of my bookmarks without issue.
It doesn't seems to have api limit problem here.

Reply:
If there is no delay, your account will be banned. I have been banned many times, so I would like users to be careful.